src/components/Discover/TrendingTokens.tsx

```diff
@@ -569,16 +572,21 @@ function SortFilter() {
   );
 }
 
+function TrendingTokensLoader() {
+  const { trending_tokens_limit } = useRemoteConfig();
+
+  return (
+    <View style={{ flex: 1 }}>
+      {Array.from({ length: trending_tokens_limit }).map((_, index) => (
+        <TrendingTokenLoadingRow key={index} />
+      ))}
+    </View>
+  );
+}
```

Comment:
we should show the correct amount of loading rows instead of the previously hardcoded 10